R/getWebServiceData.R

```diff
@@ -106,6 +107,29 @@ getQuerySummary <- function(url){
   return(retquery)
 }
 
+retryPOST <- function(obs_url, ...) {
+
+  if (nchar(obs_url) < 2048){
+    resp <- httr::RETRY("GET", 
+                      obs_url, ..., 
+                      httr::user_agent(default_ua())) 
+  } else {
+    split <- strsplit(obs_url, "?", fixed=TRUE)
+    obs_url <- split[[1]][1]
+    query <- split[[1]][2]
+    # resp <- httr::RETRY("POST",
```

Comment:
Yeah, it took me a little bit of futzing to figure out the `httr::content_type("application/json")` which is why I left it commented in there (in case I wasn't the one who would work on it down the road). But...I'll put a few notes in #434